twingraph/src/main/kotlin/com/cosmotech/twingraph/api/TwingraphServiceImpl.kt

```diff
@@ -164,6 +164,35 @@ class TwingraphServiceImpl(
     return resultSet.toJsonString()
   }
 
+  override fun updateGraphMetaData(
+      organizationId: String,
+      graphId: String,
+      requestBody: Map<String, String>
+  ): Any {
+    val organization = organizationService.findOrganizationById(organizationId)
+    csmRbac.verify(organization.getRbac(), PERMISSION_READ)
+
+    if (requestBody.containsKey("graphRotation")) {
+      val graphRotation = requestBody["graphRotation"]?.toInt()
+      if (graphRotation == null || graphRotation < 1) {
+        throw CsmClientException("GraphRotation should be a positive integer")
+      }
+    }
+
+    val metaDataKey = "${graphId}MetaData"
```

Comment:
Could you make  a String extension instead ? as we certainly use this somewhere else and will certainly reuse it